oshdb-filter/src/main/java/org/heigit/ohsome/oshdb/filter/IdFilterEqualsAnyOf.java

```diff
@@ -38,12 +44,18 @@ boolean applyOSMNegated(OSMEntity entity) {
 
       @Override
       public String toString() {
-        return "id:in" + this.ids.stream().map(String::valueOf).collect(Collectors.joining(","));
+        return "id:in" + ids.stream().map(String::valueOf).collect(Collectors.joining(","));
       }
     });
 
-    if (idList.isEmpty()) {
+    if (ids.isEmpty()) {
       throw new IllegalStateException("list of ids must not be empty in a id in (list) filter");
     }
+
+    this.ids = ids;
+  }
+
+  public Set<Long> getIds() {
```

Comment:
this could be included in `ParseTest.testIdFilterEqualsAnyOf` to check if it returns the expected values